### ISSUE TITLE
renovate-config-validator reports "INFO: Config validated successfully" when no config exists

### ISSUE THREAD

**Issue by Konafets** [renovatebot/renovate#16025]
### How are you running Renovate?

Mend Renovate hosted app on github.com

### If you're self-hosting Renovate, tell us what version of Renovate you run.

32.83.2

### Was this something which used to work for you, and then stopped?

I never saw this working

### Describe the bug

I wanted to recreate the on-boarding PR on Gitlab via PR. I did not merge the first onboarding PR, so I do not have any renovate related config file and since its a JAVA project, no `package.json`. But when I run `renovate-config-validator` it reports: 
> INFO: Config validated successfully

I expected either it tells me that there is no config file and/or offers to create one with basic settings.

### Have you created a minimal reproduction repository?

No reproduction repository

**Comment by PhilipAbed** [comment]
Hi, please elaborate, what do you mean by `renovate-config-validator`? 
and can you please send all debug logs? 
could this be related to the [requireConfig](https://docs.renovatebot.com/self-hosted-configuration/#requireconfig)?

**Comment by Konafets** [comment]
Hi, 

I installed Renovate Bot app in Github.com and gave permissions to one repo. RB created the on-boarding PR. Beside that I had another PR, which already updated some of the dependencies, RB would like to update as well. 

[Here](https://docs.renovatebot.com/getting-started/installing-onboarding/#nuke-config-and-re-onboard) I read that I should rename the onboarding PR from "Configure Renovate"  to "Configure Renovate - old" and wait that RB creates a new one. I did, but RB did not created a new one, so I keep searching and found these instructions https://docs.renovatebot.com/getting-started/installing-onboarding/#reconfigure-via-pr. I installed RB globally via NPM and executed `renovate-config-validator` with the hope that might help in terms to create an onboarding commit locally. To my surprise it told my that my configuration was validated successfully which is weird since I do not have a RB configuration file.

TLDR: The CLI app `renovate-config-validator` seems not to check if a configuration file exist in the current directory.

**Comment by PhilipAbed** [comment]
i'm sorry but i believe you should have this parameter, and thats where the config is found, `RENOVATE_CONFIG_FILE`

```
C:\tests\renovateTutorial>renovate-config-validator
DEBUG: Using RE2 as regex engine
FATAL: Custom config file specified in RENOVATE_CONFIG_FILE must exist
       "configFile": "C:\\github\\renovateWs\\config.js"
```

**Comment by PhilipAbed** [comment]
C:\tests\renovateTutorial>renovate-config-validator
DEBUG: Using RE2 as regex engine
DEBUG: Checking for config file in C:\tests\renovateTutorial/config.js
DEBUG: No config file found on disk - skipping
 INFO: Config validated successfully

**Comment by PhilipAbed** [comment]
@rarkins 
if the config file is not found we are just Merging the other default configs in, and the validation is successful anyway,
But because we are running from the script it doesn't pass through the `requireConfig` check, we do that when Onboarding pr..
so its just OK to keep going in a real repo run

**Comment by Konafets** [comment]
Maybe you could add an additional info, that the current path does not contain a configuration file.

**Comment by PhilipAbed** [comment]
we might just need to add `config.requireConfig=required` and file not found => exit code "-1" with error message maybe

**Comment by rarkins** [comment]
I think it will always be true if there's no config

**Comment by rarkins** [comment]
Resolution: fail if no config file found at all

**Comment by HonkingGoose** [comment]
I'm re-opening this issue because we reverted the PR that closed this issue:

- #23815 (PR that was supposed to close this issue)
- #25861 (PR that reverts to known-good situation)

**Comment by viceice** [comment]
I think it should only fail if the strict flag is used
